source/01-LE/02.ptx

```diff
@@ -300,8 +459,8 @@ Recall that a matrix is in <term>reduced row echelon form</term> (<term>RREF</te
 </li>
 </ol>
     <p>
-For each matrix, circle the leading terms, and label it as RREF or not RREF.
-For the ones not in RREF, find their RREF.
+For each matrix, mark the leading terms, and label it as RREF or not RREF.
+For the ones not in RREF, determine which rule is violated and how it might be fixed.
```

Comment:
Encourage discussion over calculation by having students explain the problem and potential solutions, rather than asking to write things out just yet.